packages/eslint/README.md

```diff
@@ -42,68 +42,53 @@ const eslint = require('@neutrinojs/eslint');
 
 // Usage shows default values
 neutrino.use(eslint, {
-  test: neutrino.regexFromExtensions(), // Uses extensions from neutrino.options.extensions
+  // Uses extensions from neutrino.options.extensions
```

Comment:
I've adjusted the wrapping here so the code block in the docs doesn't need to have horizontal scroll bars.